tests/chainer_tests/links_tests/model_tests/test_vision.py

```diff
@@ -12,14 +13,22 @@
 
 
 @testing.parameterize(*testing.product({
+    'dtype': [numpy.float32],
     'n_layers': [50, 101, 152],
     'downsample_fb': [True, False],
-}))
+}) + [{
+    'dtype': numpy.float16,
+    'n_layers': 50,
+    'downsample_fb': False,
+}])
 @unittest.skipUnless(resnet.available, 'Pillow is required')
 @attr.slow
 class TestResNetLayers(unittest.TestCase):
 
     def setUp(self):
+        config = chainer.config
```

Comment:
See this comment: https://github.com/chainer/chainer/pull/5114#discussion_r203917883

Reply:
Fixed!